auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -201,6 +201,10 @@ public static AuthUI getInstance() {
      */
     @NonNull
     public static AuthUI getInstance(@NonNull FirebaseApp app) {
+        if (ProviderAvailability.IS_TWITTER_AVAILABLE) {
+            Log.w(TAG, "You no longer need TwitterKit to sign in with Twitter.");
```

Comment:
Let's flesh this out.  "Beginning with FirebaseUI 6.2.0 you no longer need to include the TwitterKit SDK to sign in with Twitter.  Go to https://{whatever the release page link} is for more information"